app/src/main/java/org/mozilla/focus/session/ui/TabsPopup.kt

```diff
@@ -48,6 +48,7 @@ class TabsPopup(
         width = FrameLayout.LayoutParams.WRAP_CONTENT
         height = FrameLayout.LayoutParams.WRAP_CONTENT
         animationStyle = android.R.style.Animation_Dialog
+        binding.popupFrame.setOnClickListener { dismiss() }
```

Comment:
I think will work the same with  binding.root.setOnClickListener { dismiss() } and dat id for popupFrame will not be needed